database/index.js

```diff
@@ -4,5 +4,24 @@ const connection = mysql.createConnection(config);
 
 connection.connect();
```

Comment:
maybe add a err handling check here?

Reply:
Good catch, I will add that. Any other comments/suggestions on this branch before I merge?

Reply:
nope looks good!